test/e2e/swaps/swap-eth.spec.js

```diff
@@ -16,7 +16,12 @@ describe('Swap Eth for another Token', function () {
           amount: 2,
           swapTo: 'DAI',
         });
-        await driver.clickElement({ text: 'Review swap', tag: 'button' });
+        const reviewSwapButton = await driver.findElement(
```

Comment:
Nit: Instead of getting the element here and running assertions on it for the text and enabled status, we can _select_ for the element using those properties. That saves time and prevents race conditions within the tests (each time we ask WebDriver for something there's a round-trip involved, and properties can change while we wait).

Reply:
The same goes for the click as well. We can convert these four lines into one line, by using `clickElement` and including these details in the selector.